src/app/feedback/feedback.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
```

Comment:
If there were many imports in this module it would make sense to do this, but rather than create a module with only FeedbackDirective in it you should simply declare the FeedbackDirective in the ResourcesModule

Reply:
@paulbert  as I am adding the directive in two modules (Home module and Resource module ) I am getting the following error, could you suggest me a better way of adding it multiple modules. 
![directive_error](https://user-images.githubusercontent.com/35338384/37649998-bd6d8db2-2c5b-11e8-97b3-542783fff4d9.PNG)

Reply:
Right, forgot about that part.  I still do not think it's good to have such a small module, so let's add this directive to the DialogsModule instead.